build/link-wrapper.sh

```diff
@@ -32,6 +35,9 @@ case $1 in
 		    $CC $OPTIONS
 		fi
 		;;
+		FreeBSD)
+			clang++ -std=c++11 -stdlib=libc++ -msse4.2 -Wno-error=unused-command-line-argument -Wno-undefined-var-template -Wno-unknown-warning-option $OPTIONS -lm -lc++ -lpthread -lexecinfo
```

Comment:
I'm assuming this should be something like

$CXX $OPTIONS

and any extra `-l` libs needed should be in _LIBS and all the other flags should be in CFLAGS, both which which should get included with ${OPTIONS}

Reply:
Ok, for this one generally when using "gmake" in FreeBSD for these types of builds the arrangement of the libs has to appear after -o and the bin location. Additionally bash has no idea what CC is (what fun right?). I have separated these into the requested bash vars for now, you'll see this with the next check in.